docs/source/release_notes/release-notes.rst

```diff
@@ -43,6 +43,22 @@ develop
 
 .. replace this with the release date and the above with v<tag> (the v makes the permalinks nice)
 
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Type
+         - Change
+    
+    *    -
```

Comment:
I don't think this should be here

Reply:
Seems like we need this empty row to prevent this error: `Insufficient data supplied (1 row(s)); no data remaining for table body, required by "list-table" directive.`